styleguide/src/Forms/Input/Input.tsx

```diff
@@ -74,8 +142,13 @@ const InputComponent = (
           disabled={disabled}
           readOnly={readOnly}
           className={inputClass}
```

Comment:
Apenas uma sugestão, mas que vai dar bem mais trabalho, porém ficaria mais simples o gerenciamento:
Em volta desse input, colocar uma div que vai receber esse `inputClass` (com a borda e etc), e dentro dessa div colocar esse start e end.
Dessa forma seria preciso apenas libar com o background e border-color nessa div e não precisaria mexer nos 3 elementos (start, end e input).
Daria também para usar essa função para o focus continuar funcionando:
https://tailwindcss.com/docs/hover-focus-and-other-states#focus-within
(mas é apenas sugestão mesmo, pois fazer isso iria dar trabalho e teria que mudar praticamente tudo)

Reply:
boa, não sabia dessa classe do tailwind, acho bem válido a gente matar aquela função de focus